2022q4/Makefile

```diff
@@ -8,34 +8,34 @@ YEAR?=	2022
 QUARTER?=	4
 
 # Entries from the various official and semi-official teams
-TeamReports= freebsd-foundation.adoc releng.adoc clusteradm.adoc ci.adoc portmgr.adoc
+TeamReports= freebsd-foundation.adoc releng.adoc clusteradm.adoc ci.adoc portmgr.adoc status.adoc
 
 # Projects that span multiple categories, from the kernel, and userspace to
 # the Ports Collection or external projects
-Projects=
+Projects= cloud-init.adoc vessel-status-report.adoc NFS-in-prisons.adoc
```

Comment:
https://github.com/freebsd/freebsd-quarterly/pull/557 brings updates for OpenStack

if there are any other cloud related projects, we can put them under their own header, rather than under ports?

Reply:
I have looked into past quarters and I have seen that we had other reports that could have gone in a cloud category: we had reports about Azure and about ENA.

I think it is a good idea to create a cloud category. I will move cloud-init.adoc and openstack-on-freebsd.adoc into the new category next time I update the Makefile.